test/distribution/test_distribution_output_shapes.py

```diff
@@ -141,6 +142,13 @@
             (3, 4),
             (5,),
         ),
+        (
+            PoissonOutput(),
+            mx.nd.random.normal(shape=(3, 4, 5, 6)),
```

Comment:
Since Poisson has the set of natural number as support, it's better to draw from something other than Normal here: you could use `mx.nd.random.poisson`, see https://mxnet.incubator.apache.org/api/python/docs/api/ndarray/random/index.html#mxnet.ndarray.random.poisson